tools/cli/go-whisk-cli/commands/util.go

```diff
@@ -61,6 +61,17 @@ func (qualifiedName QualifiedName) String() string {
     return strings.Join(output, "")
 }
 
+// replaceMissingLeadSlash(name) returns a (possibly fully qualified) resource name,
+//      inserting a leading '/' if it is of 3 parts (namespace/packageName/actionName)
+//      and lacking the leading '/'.
+func replaceMissingLeadSlash(name string) string {
```

Comment:
Would it be more clear if this were a method on the `QualifedName` struct?  Perhaps rename it as well to a name that implies it only really works on a fully qualified entity name